cmd/snap/cmd_userd.go

```diff
@@ -0,0 +1,154 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2017 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/jessevdk/go-flags"
+
+	ifaces "github.com/snapcore/snapd/dbus"
+	"github.com/snapcore/snapd/i18n"
+	"github.com/snapcore/snapd/logger"
+
+	"github.com/godbus/dbus"
+	"github.com/godbus/dbus/introspect"
+
+	"gopkg.in/tomb.v2"
+)
+
+const (
+	busName  = "io.snapcraft.SafeLauncher"
+	basePath = "/"
```

Comment:
you normally used /io/snapcraft/SafeLauncher, yes. This is left over from when I changed the bus name to do the transition away from com.canonical.SafeLauncher. Let me change the path.